src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -29,7 +29,7 @@ import { coalesce } from 'vs/base/common/arrays';
 import { isCodeEditor, isDiffEditor, ICodeEditor, IDiffEditor } from 'vs/editor/browser/editorBrowser';
 import { IEditorGroupView, IEditorOpeningEvent, EditorGroupsServiceImpl, EditorServiceImpl } from 'vs/workbench/browser/parts/editor/editor';
 import { IUriDisplayService } from 'vs/platform/uriDisplay/common/uriDisplay';
-import { convertEditorInput } from 'sql/parts/common/customInputConverter';
+import { convertEditorInput, notebookModeId } from 'sql/parts/common/customInputConverter';
```

Comment:
Note: please add the ``{{ SQL CARBON EDIT }}` tag above this, it was missed in last merge.
Also, you can either import the INotebookService here or put logic for detecting correct mode (sql vs. notebook) in the customInputConverter. I can see that making the most sense.